### ISSUE TITLE
Fail on windows

### ISSUE THREAD

**Issue by at15** [arrowrowe/es6-playground#2]
error. I am using git bash

node v0.12.0 Windows8.1

``` sh

$ npm install -g traceur
C:\Users\Pillar\AppData\Roaming\npm\traceur -> C:\Users\Pillar\AppData\Roaming\n
pm\node_modules\traceur\traceur
traceur@0.0.90 C:\Users\Pillar\AppData\Roaming\npm\node_modules\traceur
├── commander@2.6.0
├── semver@2.3.2
├── rsvp@3.0.18
├── glob@4.3.5 (inherits@2.0.1, once@1.3.2, inflight@1.0.4, minimatch@2.0.8)
└── source-map-support@0.2.10 (source-map@0.1.32)

$ git clone git@github.com:arrowrowe/es6-playground.git
$ cd es6-playground/

$ ./try.sh try-string.js
\033[1;34mTranspiling \033[0;37mtry-string.js\033[1;34m to \033[0;37mout/try-str
ing.js\033[1;34m...
Transpiled.
Calling Node...
=======================\033[0m
module.js:338
    throw err;
          ^
Error: Cannot find module '/usr/local/lib/node_modules/traceur/bin/traceur-runti
me'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (d:\github-other\es6-playground\try.js:1:63)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
\033[1;34m=======================
End.
```

**Comment by at15** [comment]
however,run `traceur try-string.js` works fine

**Comment by at15** [comment]
ok... I got it, `try.js`  `require` a fixed path `require('/usr/local/lib/node_modules/traceur/bin/traceur-runtime')`

**Comment by arrowrowe** [comment]
My bad. Wanted to output to files in case we need to check the es5 equivalents.
`traceur some-es6-code.js` works fine.

**Comment by at15** [comment]
already fixed in fcf70d1be420e530f3a6fe3f051ffeaaf7fd6aba
